pkg/appgw/internaltypes_test.go

```diff
@@ -107,25 +109,25 @@ var _ = Describe("Test string key generators", func() {
 			Expect(len(actual)).To(Equal(80))
 		})
 		It("generateProbeName preserves keys in 80 charaters of length or less", func() {
-			expected := "k8s-ag-ingress-xxxxxx-yyyyyy-pb-zzzz"
+			expected := "xxxxxx-yyyyyy-pb-zzzz"
```

Comment:
append `agPrefix +` ?

Reply:
We could have kept it in this test - let me add it back.